universal-application-tool-0.0.1/app/services/program/BlockDefinition.java

```diff
@@ -87,6 +87,13 @@ public boolean isRepeated() {
     return repeaterId().isPresent();
   }
 
+  public QuestionDefinition getEnumerationQuestionDefinition() {
+    if (isRepeater()) {
+      return getQuestionDefinition(0);
+    }
+    throw new RuntimeException("Only a repeater block can have an enumeration question definition.");
```

Comment:
nit/question: Should we say "Only a **enumeration** block can.."? Or should we save that update till we rename the methods for like `isEnumeration` etc?

Reply:
I think we save this for one big PR later.